src/main/java/org/mule/tools/devkit/sonar/checks/git/GitCheck.java

```diff
@@ -0,0 +1,10 @@
+package org.mule.tools.devkit.sonar.checks.git;
+
+import org.sonar.api.batch.SensorContext;
+import org.sonar.api.batch.fs.InputFile;
+import org.sonar.api.resources.Project;
+
+@FunctionalInterface
+public interface GitCheck {
+    void analyse(Project project, SensorContext context, InputFile inputFile);
```

Comment:
analyze = British, analyze = American
I'm fine with either. Just saying this to keep everything consistent. :-)